spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :feature do
+  before do
+    @user = create(:user, :with_posts)
+    @most_recent_posts = @user.posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] You have done an excellent job, however, you forgot also to add tests for the user show page:
````
- I can see a button that lets me view all of a user's posts.
- When I click a user's post, it redirects me to that post's show page.
````
Kindly add them as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20a%20button,to%20that%20post%27s%20show%20page.) 😄